File: nahomy-c2-backend/src/infrastructure/config/RabbitMQConfig.ts
export class RabbitMQConfig {
  constructor(
    readonly url: string = process.env.RABBITMQ_URL || 'amqp://localhost:5672',
    readonly exchangeName: string = process.env.RABBITMQ_EXCHANGE || 'c2.events',
    readonly intentionQueue: string = 'c2.intention.queue',
    readonly auditQueue: string = 'c2.audit.queue',
    readonly metricsQueue: string = 'c2.metrics.queue',
  ) {}
}

File: nahomy-c2-backend/src/infrastructure/config/rabbitmq_config.py
import os


def get_rabbitmq_url() -> str:
    return os.getenv("RABBITMQ_URL", "amqp://guest:guest@localhost:5672")


RABBITMQ_EXCHANGE = os.getenv("RABBITMQ_EXCHANGE", "c2.events")
INTENTION_QUEUE = "c2.intention.queue"
AUDIT_QUEUE = "c2.audit.queue"
METRICS_QUEUE = "c2.metrics.queue"
